src/functions/initializeActivities.spec.ts

```diff
@@ -161,18 +168,18 @@ function createExpected(currentDate: number, activities: any) {
     return expected;
 }
 
-function createEvent() : any {
+function createEvent(): any {
     return {
         uid: '123',
         stages: {
-            Stage1:''
+            Stage1: ''
         },
         metadata: {
-            company:"0",
-            effectiveDate:"1/2/2018",
-            lineOfBusiness:'Personal',
-            policies:[{policyId: "00283316-d954-74f6-de8b-1f72b9b58e66"}],
-            riskState:'ID',
+            company: "0",
+            effectiveDate: "1/2/2018",
+            lineOfBusiness: 'Personal',
+            policies: [{ policyId: "00283316-d954-74f6-de8b-1f72b9b58e66" }],
+            riskState: 'ID',
             workflow: 'test'
         }
     };
```

Comment:
We're missing merge unit tests for workflows which are already partially completed, or which have components in error states.

New sections of unit tests should cover
- InitializeActivities receives an existing WorkflowStatus for the uid, Doesn't recieve an existing workflow id, metadata doesn't match, etc
- GetActivityForStage (New Activity, Reset existing activity for error states, in-progress states, etc)